opm/autodiff/NewtonIterationBlackoilInterleaved.cpp

```diff
@@ -120,23 +182,39 @@ namespace Opm
             typedef std::unique_ptr<typename ScalarProductChooser::ScalarProduct> SPPointer;
             SPPointer sp(ScalarProductChooser::construct(parallelInformation_arg));
 
-            // Construct preconditioner.
-            auto precond = constructPrecond(opA, parallelInformation_arg);
-
             // Communicate if parallel.
             parallelInformation_arg.copyOwnerToAll(istlb, istlb);
 
-            // Solve.
-            solve(opA, x, istlb, *sp, *precond, result);
-        }
+#if ! HAVE_UMFPACK
+            const bool useAmg = false ;
+            if( useAmg )
```

Comment:
This is still testing. Also, the selection of preconditioners should be a parameter and not hard coded. This change should be done when the Linear solver interface is revised and the PETSc stuff is merged. I wanted to do that but my project owner wants the focus on performance.